test/test_torch.py

```diff
@@ -2070,8 +2070,8 @@ def test_inference(default_dtype):
         test_inference(torch.float64)
         test_inference(torch.float32)
 
-    def test_qengnie(self):
-        qengines = torch.backends.quantized.get_supported_qengines()
+    def test_qengine(self):
+        qengines = torch.backends.quantized.supported_engines
```

Comment:
supported_qengines?

Reply:
there's already "quantized" in the name, so I simplified the name

Reply:
we have `supportedQEngine` in c++, I think it's better to align the names.. maybe change both of them either to `qengine` or `engine`, I think supported_qengine is better since `QEngine` is one word.. just like `QScheme`